cli/build/register.ts

```diff
@@ -207,13 +216,10 @@ export const registerBuild = (program: Command) => {
           if (options?.site) {
             const indexHtml = getStaticIndexHtmlFile({
               files: staticFileReferences,
-              standaloneScriptSrc: "./standalone.min.js",
+              standaloneScriptContent: runFrameStandaloneBundleContent,
+              filesWithData: staticFilesWithData,
```

Comment:
data is a banned word

```suggestion
              fsMap: staticFsMap,
```